docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       DB_PASS: root
       DB_USER: root
       DB_HOSTNAME: db
-      RACK_ENV: development
```

Comment:
This should be unchanged. We only use docker-compose locally on the dev machine and we would like the container to run Rails in development mode